src/main/resources/r/run_pathways.R

```diff
@@ -0,0 +1,163 @@
+library(DatabaseConnector)
+library(SqlRender)
+library(rlist)
+library(purrr)
+library(dplyr)
+
+getEventCohortCodes <- function(pathwayAnalysis) {
```

Comment:
THe R script should be as simple as the Cohort Characterization one. Seems like we need to extract Pathways logic to a separate R package